main.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+    $instaobj = Instamojo\Instamojo::init('app',[
+        "client_id" =>  $_ENV["CLIENT_ID"],
+        "client_secret" => $_ENV["CLIENT_SECRET"]
+       
+    ],true);
+ 
+    
+    // $transaction_id = "TEST_".time();
+    // var_dump([
+    //     "name" => "XYZ",
+    //     "email" => "xyz@squareboat.com",
+    //     "phone" => "9999999988",
+    //     "amount" => 200,
+    //     "transaction_id" => $transaction_id,
+    //     "currency" => "INR"
+    // ]);
+
+    $payment_request = $instaobj->createPaymentRequest([
+        'amount'=>10,
+        'purpose'=>"Test script"
+        ]);
+        var_dump($payment_request['id']);
+        try{
```

Comment:
Please use single quotes instead of double quotes for consistent style & code efficiency